vnext/Microsoft.ReactNative/ReactNativeApplication.idl

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import "ReactNativeHost.idl";
+import "ReactNativeIsland.idl";
+
+namespace Microsoft.ReactNative
+{
+#ifdef USE_WINUI3
+    struct ReactApplicationInstanceSettings
+    {
+        String bundleRootPath;
+    };
+
+    runtimeclass ReactNativeApplication
```

Comment:
We already have `IReactApplication`, Can we avoid adding another application type and reuse the existing one?